pep-0696.rst

```diff
@@ -2,13 +2,15 @@ PEP: 696
 Title: Type defaults for TypeVarLikes
 Author: James Hilton-Balfe <gobot1234yt@gmail.com>
 Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
-Discussions-To: typing-sig@python.org
+Discussions-To: https://mail.python.org/archives/list/typing-sig@python.org/thread/7VWBZWXTCX6RAJO6GG67BAXUPFZ24NTC
 Status: Draft
 Type: Standards Track
 Topic: Typing
 Content-Type: text/x-rst
 Created: 14-Jul-2022
 Python-Version: 3.12
+Post-History: `22-Mar-2022 <https://mail.python.org/archives/list/typing-sig@python.org/thread/7VWBZWXTCX6RAJO6GG67BAXUPFZ24NTC/>`__,
+              `08-Jan-2023 <https://discuss.python.org/t/pep-696-type-defaults-for-typevarlikes/22569/5/>`__,
```

Comment:
The reason the linter (correctly) flagged this was due to it accidentally being a direct link to an arbitrary post down in the thread, instead of the thread itself. You were able to unintentionally bypass it due to the escape hatch of manually adding a trailing slash (which is not added by Discourse by default, like the post number is) recently implemented in #3005 in response to an author request in #3004 , which allows direct links to specific posts in special cases where the author specifically finds it necessary and intends to do so.

To actually fix this properly, the trailing post number, `5/`, should be removed from this URL so it goes to the thread, not an arbitrary post within it.

Reply:
I've opened #3053 to fix this.